L1Trigger/DTTriggerPhase2/plugins/DTTrigPhase2Prod.cc

```diff
@@ -900,9 +1251,21 @@ void DTTrigPhase2Prod::fillDescriptions(edm::ConfigurationDescriptions& descript
   desc.add<int>("scenario", 0);
   desc.add<int>("df_extended", 0);
   desc.add<int>("max_primitives", 999);
+  desc.add<bool>("output_mixer", false);
+  desc.add<bool>("output_latpredictor", false);
+  desc.add<bool>("output_slfitter", false);
+  desc.add<bool>("output_slfilter", false);
+  desc.add<bool>("output_confirmed", false);
+  desc.add<bool>("output_matcher", false);
   desc.add<edm::FileInPath>("ttrig_filename", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/wire_rawId_ttrig.txt"));
   desc.add<edm::FileInPath>("z_filename", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/wire_rawId_z.txt"));
+  desc.add<edm::FileInPath>("lut_sl1", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/fitterlut_sl1.dat"));
+  desc.add<edm::FileInPath>("lut_sl2", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/fitterlut_slx.dat"));
+  desc.add<edm::FileInPath>("lut_sl3", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/fitterlut_sl3.dat"));
+  desc.add<edm::FileInPath>("lut_2sl", edm::FileInPath("L1Trigger/DTTriggerPhase2/data/fitterlut_2sl.dat"));
```

Comment:
are these already present in the data externals? Or has a PR been opened?

Reply:
As explained in the PR description: https://github.com/cms-data/L1Trigger-DTTriggerPhase2/pull/3